README.md

```diff
@@ -233,6 +233,20 @@ Same as `sanitize(fields)`, but only sanitizing `req.query`.
 
 ---
 
+## Wildcards (*)
+To validate or sanitize an array of strings use wildcards(*).
+
+```js
+// Wildcards * are accepted!
```

Comment:
Add comment saying that the `postalCode` of every item under `addresses` will be validated?

Reply:
oops.